docarray/typing/tensor/audio/audio_torch_tensor.py

```diff
@@ -1,12 +1,14 @@
 from typing import TypeVar
 
+from docarray.typing.proto_register import register_proto
 from docarray.typing.tensor.audio.abstract_audio_tensor import AbstractAudioTensor
 from docarray.typing.tensor.audio.audio_ndarray import MAX_INT_16
 from docarray.typing.tensor.torch_tensor import TorchTensor, metaTorchAndNode
 
 T = TypeVar('T', bound='AudioTorchTensor')
 
 
+@register_proto(proto_type_name='audio_torch_tensor')
```

Comment:
the proto type should not be `ndarrayTensor`? and nothing related to `audio`?

Reply:
but you need to store the information that this is a AudioTensor right ?

Reply:
okey I understand now